pyanaconda/modules/subscription/subscription.py

```diff
@@ -355,6 +360,32 @@ def set_subscription_request(self, subscription_request):
         self.subscription_request_changed.emit()
         log.debug("A subscription request set: %s", str(self._subscription_request))
 
+    @property
+    def attached_subscriptions(self):
+        """A list of attached subscriptions.
+
+        The list holds DBus structures with each structure holding information about
+        one attached subscription. A system that has been successfully registered and
+        subscribed usually has one or more subscriptions attached.
+
+        :return: list of DBus structures, one per attached subscription
+        :rtype: list of AttachedSubscription instances
+        """
+        return self._attached_subscriptions
+
+    def set_attached_subscriptions(self, attached_subscriptions):
```

Comment:
Shouldn't this be a private method if it is not an API of this class?

Reply:
We have it like this in a couple places for other properties & I think I saw something similar in other modules. So I think such a change is more suitable for a followup PR.